### DISCUSSION TITLE
Hoppscotch Desktop App

### DISCUSSION THREAD

**Discussion by rafiandria23** [hoppscotch/hoppscotch#2854] [Ideas]
Hi, I'm Adam.

I'm just wondering... does Hoppscotch have a desktop app installer? If not I'm interested in developing it using Electron to make it cross-platform. Let's discuss this further. Thank you!

**Comment by AndrewBastin** [comment]
Hi @rafiandria23! Thank you for your interest!

Yes, the desktop app is something we are actively working on. But we are not interested in building our app in Electron because that is literally against our founding statements. We hope to release the native app sometime next year.

We have news to share about this hopefully by the end of the year, stay tuned to our communication channels!

**Reply by rafiandria23** [reply]
Well noted, @AndrewBastin

Thank you!

**Reply by sistematico** [reply]
Any news?

**Reply by nsde** [reply]
I'm also interested. You could use Neutralinojs or Tauri.

**Reply by liyasthomas** [reply]
_Thanks for being patient while we built the initial alpha version of the Hoppscotch Desktop App._

## Introducing Hoppscotch Desktop App

Hoppscotch Desktop App is a cross-platform desktop application built with Tauri and Hoppscotch Web Client. It is a standalone version of Hoppscotch that can be installed on your computer and used without a browser. Hoppscotch Desktop App is currently in alpha and is available for Mac, Windows, and Linux.

[<kbd> <br> Download Hoppscotch Desktop App ➜ <br> </kbd>](https://hoppscotch.com/download)

[Read blog](https://hoppscotch.com/blog/introducing-hoppscotch-desktop-application) | [Read changelog](https://hoppscotch.com/changelog/08-11-2023)

<sub>_The Hoppscotch Desktop App is currently in alpha. This means that it is not yet feature-complete and may contain bugs. We are actively working on improving the app and adding new features. You may encounter bugs and issues while using the app. If you do, please report them on our [GitHub repository](https://github.com/hoppscotch/hoppscotch)._</sub>
